VimeoNetworking/Sources/Models/Spatial.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Spatial.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 1/6/17.
+//
+//
+//
+
+/// Spatial stores all information related to threesixty video
+public class Spatial: VIMModelObject
+{
+    
+    /// Represents the projection. Value returned by the server can be: "equirectangular", "cylindrical", "cubical", "pyramid", "dome".
+    dynamic var projection: String?
```

Comment:
that was the idea, but finally i don't think we need to kvo this. I'll update the PR.